gcp/sql.tf

```diff
@@ -136,9 +140,33 @@ resource "google_sql_database" "approvalq" {
   instance = google_sql_database_instance.master.name
 }
 
+resource "google_sql_database" "inventory" {
```

Comment:
We also have a script that creates the dbs https://github.com/smallstep/gitops/blob/main/env/replicated/admin-tools/create-dbs-configmap.yaml#L10